src/WasmComponent.Sdk/build/WasmComponent.Sdk.targets

```diff
@@ -31,13 +31,14 @@
             <WasiPreview1AdapterType>reactor</WasiPreview1AdapterType>
             <WasiPreview1AdapterType Condition="'$(OutputType.ToLower())' == 'exe'">command</WasiPreview1AdapterType>
             <WasiPreview1AdapterPath>$(MSBuildThisFileDirectory)../tools/wasi-wasm/wasi_snapshot_preview1.$(WasiPreview1AdapterType).wasm</WasiPreview1AdapterPath>
-            <NativeComponentBinary>$(NativeOutputPath)$(TargetName).component.wasm</NativeComponentBinary>
+            <!-- NativeComponentBinary should have a filename in kebab-case -->  
+            <NativeComponentBinary>$(NativeOutputPath)$(TargetName.ToLower().Replace(" ", "-"))-component.wasm</NativeComponentBinary>
```

Comment:
Thanks for submitting this PR. It looks great!

Could you clarify why you think the output should use kebab-case? Is that part of a spec somewhere, or is it just your view about what most people are doing?

Reply:
The kebab-case was a surprise to me as well. Because in the version you used initially, this restriction is not yet active. However, if you would use `wasm-tools compose -o output.wasm input.wasm -d dependency.component.wasm` (see the dot before component instead of the hyphen) then you are presented with an error "error: `dependency.component` is not in kebab case (at offset 0x0)"

In the documentation the kebab-case is only specified for world identifiers, and not for the files you embed those worlds in.

Reply:
OK, thanks! Personally I wish the WASI ecosystem would avoid making assumptions like this but recognize it's just something we have to deal with right now. I appreciate you figuring it out.

Reply:
> The kebab-case was a surprise to me as well. Because in the version you used initially, this restriction is not yet active. However, if you would use `wasm-tools compose -o output.wasm input.wasm -d dependency.component.wasm` (see the dot before component instead of the hyphen) then you are presented with an error "error: `dependency.component` is not in kebab case (at offset 0x0)"
> 
> In the documentation the kebab-case is only specified for world identifiers, and not for the files you embed those worlds in.

Yes. I got the same problem, and had to rename it to kebab-case for the component, then we are fine to compose it.